ServiceBoxWeb/src/main/java/com/infoshareacademy/controller/employee/ServiceOrderController.java

```diff
@@ -15,7 +14,7 @@
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
```

Comment:
Blank line - should be deleted?

Reply:
No comment, everything was already corrected.